cultcargo/bdsf.yml

```
@@ -107,9 +115,7 @@ cabs:
         dtype: bool
         default: false
       src_ra_dec:
-        dtype: List[Tuple]
-      src_radius_pix:
-        dtype: int
+        dtype: List[Tuple[int]]
```

Also, the `tuple` items here are ra and dec in degrees, i.e. their data type has to be `float`.
I wonder if this supposed to be `dtype: List[Tuple[float,float]]`?

```suggestion
        dtype: List[Tuple[float,float]]
```